site/docs/components/static-list/examples.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+  <LivePreview componentName="static-list" exampleName="DefaultSingle" >
+
+## Default Single
+
+Default `Static list` with a single item.
+
+  </LivePreview>
+
+  <LivePreview componentName="static-list" exampleName="DefaultMultiple" >
+
+## Default Multiple
+
+Default `Static list` group showing multiple items stacked together. 
+
+  </LivePreview>
+
+  <LivePreview componentName="static-list" exampleName="AdditionalLabel" >
+
+  ## Additional label
+
+`Static list` items with additional label  grouped together to organize related content.
+  </LivePreview>
+
+  <LivePreview componentName="static-list" exampleName="WithIcons" >
+
+## With Icons
+
+`Static list` items with descriptive icon  grouped together to organize related content. You can change the default descriptive icon with other icons based on your use-case.
+
+  </LivePreview>
+
+  <LivePreview componentName="static-list" exampleName="WithButtons" >
+
+## With Buttons
+
+`Static List `items with interactive actions. You can change the default button based on your use-case.
```

Comment:
Should the design guide be written more in wording? e.g. recommend icon on the left, actions on the right sort of angle?

"You can change the default button based on your use-case" feels a little redundant. Anything can be changed based on use case. If we're talking about button variants, then be more specific?